pluck_test.go

```diff
@@ -12,6 +12,23 @@ func ExamplePluck() {
 	// Output: [[<nil>] [4]]
 }
 
+func TestPluckEmptyKeys(t *testing.T) {
+	output := Pluck(
+		[]map[string]any{
```

Comment:
Just out of curiosity: Is there any advantage of using `any` instead of `int` here?

Reply:
@cuducos not really afaik